package.json

```diff
@@ -12,11 +12,13 @@
     "@types/react-dom": "^18.0.6",
     "@types/styled-components": "^5.1.26",
     "axios": "^1.2.1",
+    "formik": "^2.2.9",
     "jest-environment-jsdom": "^29.1.2",
     "jest-styled-components": "^7.1.1",
     "json-server": "^0.17.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-hook-form": "^7.42.1",
```

Comment:
there's no need to have 2. You can choose either formik or react-hook-form and uninstall the other one